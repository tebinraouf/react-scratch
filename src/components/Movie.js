import React from "react";
import PropTypes from "prop-types";

const Movie = (props) => {
	const match = props.match === undefined ? {} : props.match;
	var id = match === undefined ? 0 : match.params;

	if (typeof id === "object") {
		id = id.id;
	}

	// const { id } = props.match.params;

	return (
		<div>
            Hello {id}
		</div>
	);
};

Movie.propTypes = {
	match: PropTypes.object
};

export default Movie;