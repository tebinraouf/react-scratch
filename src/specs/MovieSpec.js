import React from "react";
import Movie from "../components/Movie";

import renderer from "react-test-renderer";

describe("<Movie />", () => {
	test("Should render correctly", () => {
		const component = renderer.create(<Movie />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});