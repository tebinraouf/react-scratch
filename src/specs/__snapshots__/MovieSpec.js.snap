// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Movie /> Should render correctly 1`] = `
<div>
  Hello 
</div>
`;
